### PR TITLE
remove recording rules

### PR DESCRIPTION
**Pull request by AjayTripathy**
Requires testing. Removes relabel rules that created backwards compatibility. Requires https://github.com/kubecost/cost-model/pull/881

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -570,9 +516,6 @@ serviceAccount:
   create: true # Set this to false if you're bringing your own service account.
   annotations: {}
   # name: kc-test
-awsstore:
-  useAwsStore: false
```
Comment (dwbrown2) [inline cost-analyzer/values.yaml:574]: should this be included?

File: cost-analyzer/values.yaml [hunk 2]
```diff
@@ -365,60 +365,6 @@ prometheus:
     enabled: false
     persistentVolume:
       enabled: true
-  serverFiles:
```
Comment (dwbrown2) [inline cost-analyzer/values.yaml:368]: I just want to confirm that none of these are needed? How have we test?

**Comment by AjayTripathy** [comment]
This is a draft I'm leaving here to test our cadvisor rules
changes/recording rules deprecation

On Mon Aug 2, 2021, 04:03 AM GMT, Webb Brown ***@***.***>
wrote:

***@***.**** commented on this pull request.
------------------------------

In cost-analyzer/values.yaml
<https://github.com/kubecost/cost-analyzer-helm-chart/pull/1006#discussion_r680639356>
:

> @@ -570,9 +516,6 @@ serviceAccount:
   create: true # Set this to false if you're bringing your own service account.
   annotations: {}
   # name: kc-test
-awsstore:
-  useAwsStore: false

should this be included?
------------------------------

In cost-analyzer/values.yaml
<https://github.com/kubecost/cost-analyzer-helm-chart/pull/1006#discussion_r680639435>
:

> @@ -365,60 +365,6 @@ prometheus:
     enabled: false
     persistentVolume:
       enabled: true
-  serverFiles:

I just want to confirm that none of these are needed? How have we test?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/kubecost/cost-analyzer-helm-chart/pull/1006#pullrequestreview-719770891>,
or unsubscribe

**Comment by AjayTripathy** [comment]
deleted in favor of https://github.com/kubecost/cost-analyzer-helm-chart/pull/1185

**Pull request closed without merging**